### PR TITLE
ss-52: add reinstall command

### PR DESCRIPTION
**Pull request by Evcat6**
Added reinstall command to package.json

### CODE REVIEW

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]
We should remove all node_modules and after run npm install

File: package.json [hunk 1]
```diff
@@ -22,7 +22,10 @@
         "lint:css": "npm run lint:css --workspaces --if-present",
         "lint:prettify": "prettier --check \"**/*.{ts,tsx,json,md,scss,html,yml}\"",
         "lint": "npm run lint:editor && npm run lint:fs && npm run lint:prettify && npm run lint:type && npm run lint:js && npm run lint:css",
-        "prettify": "prettier --write \"**/*.{ts,tsx,json,md,scss,html,yml}\""
+        "prettify": "prettier --write \"**/*.{ts,tsx,json,md,scss,html,yml}\"",
+        "remove:node_modules:backend": "npm run remove:node_modules -w backend",
+        "remove:node_modules:frontend": "npm run remove:node_modules -w frontend",
+        "reinstall": "npm run remove:node_modules:backend && npm run remove:node_modules:frontend && rm -rf node_modules && npm install"
```
Comment (nikita-remeslov) [inline package.json:28]: `shared` folder
and if you add separate scrip for remove and install. You should make the same for the root folder.

Comment (nikita-remeslov) [reply package.json:28]: and also we can simplify it

`"clear:backend": "cd backend && rm -rf node_modules",`
`"clear:frontend": "cd frontend && rm -rf node_modules",`
`"reinstall": "npm run clear:backend && npm run clear:frontend && npm install",`

Comment (Evcat6) [reply package.json]: Resolved in [e4517d3](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/54/commits/e4517d390135010328a0b8741b7c23bb2b86bb83)

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**